core/cc/connection_test.cpp

```diff
@@ -21,13 +21,13 @@
 
 #include <string>
 
+using ::testing::_;
```

Comment:
I'm concerned that your clang-format is doing something different to my clang-format here!
Sure this was with the Google style?

Reply:
I got clang-format via `apt-get install clang-format`. It shows the following version:
```
$ clang-format -version
clang-format version 6.0.0-1ubuntu2 (tags/RELEASE_600/final)
```
I've also built it at head and it makes the same changes (I compared against running it on clean master). What's the version of your clang-format?

> Sure this was with the Google style?

Yes, without `-style=Google` it would make a whole ton more changes...